tests/test_events.py

```diff
@@ -1,14 +1,16 @@
 import pytest
 
+from tests.utils.syncify import run_sync
 from tests.utils.fixtures.mock_event import MockEvent
 from workos.events import AsyncEvents, Events
 
 
+@pytest.mark.sync_and_async({"sync": Events, "async": AsyncEvents})
```

Comment:
I guess this doesn't take kwargs?

Reply:
I'm pretty sure it does actually, I should just do that. Updating!